.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: objective-c
+osx_image: xcode9
+cache:
+  - bundler
+  - cocoapods
+rvm:
+  - 2.2.2
+
+env:
+  global:
+    - WORKSPACE="LonaStudio.xcworkspace"
+    - OSX_APP_SCHEME="LonaStudio"
+    - OSX_SDK=macosx10.13
+  matrix:
+    - DESTINATION="arch=x86_64"                      SCHEME="$OSX_APP_SCHEME"           SDK="$OSX_SDK"     RUN_TESTS="NO"    BUILD_EXAMPLE="YES"
+    - DESTINATION="arch=x86_64"                      SCHEME="$OSX_APP_SCHEME"           SDK="$OSX_SDK"     RUN_TESTS="YES"    BUILD_EXAMPLE="NO"
```

Comment:
you can make this a bit shorter by omitting the "no" entirely; and you may want to put "RUN_TESTS" and "BUILD_EXAMPLE" first, so it's clearer in the travis UI what's happening.